fe/fe-core/src/main/java/org/apache/doris/planner/PredicatePushDown.java

```diff
@@ -49,6 +49,13 @@ public static PlanNode visitScanNode(ScanNode scanNode, JoinOperator joinOp, Ana
     }
 
     private static void predicateFromLeftSidePropagatesToRightSide(ScanNode scanNode, Analyzer analyzer) {
+        List<Expr> unassignedRightSideConjuncts = analyzer.getUnassignedConjuncts(scanNode);
```

Comment:
This function is used to propagate left predicate to the right.
It seems that the hive table can also support this rule.
Why should It return early?